commands/cmd-restore/src/lib.rs

```diff
@@ -33,6 +33,10 @@ impl OroCommand for RestoreCmd {
             .root
             .expect("root should've been set by global defaults");
         let mut nm = NodeMaintainerOptions::new();
+        #[cfg(not(target_arch = "wasm32"))]
+        {
+            nm = nm.progress_bar(true);
+        };
```

Comment:
you don't need to worry about this one. cmd-restore is desktop-only, never wasm32